### PR TITLE
kernel: config: enable overlay_fs for DinK use case

### PR DESCRIPTION
**Pull request by egernst**
To run Docker within a Kata container, OVERLAY_FS needs to be enabled.

Fixes: #490

Signed-off-by: Eric Ernst <eric.ernst@intel.com>

### CODE REVIEW

File: kernel/configs/x86_64_kata_kvm_4.19.x [hunk 1]
```diff
@@ -2600,7 +2600,13 @@ CONFIG_AUTOFS_FS=y
 CONFIG_FUSE_FS=y
 # CONFIG_CUSE is not set
 CONFIG_VIRTIO_FS=y
-# CONFIG_OVERLAY_FS is not set
+CONFIG_OVERLAY_FS=y
+CONFIG_OVERLAY_FS_REDIRECT_DIR=y
```
Comment (grahamwhaley) [inline kernel/configs/x86_64_kata_kvm_4.19.x]: Just to check - did you explicitly choose these settings, or take the menuconfig defaults?
Do we know we need/want `REDIRECT_DIR` for instance?

**Comment by grahamwhaley** [comment]
Ah, I see we already do have a Dep check over at https://github.com/kata-containers/runtime/pull/1634 - you get extra bonus points for that @egernst  ;-)

File: kernel/configs/x86_64_kata_kvm_4.19.x (hunk 1 above)
Comment (egernst) [reply kernel/configs/x86_64_kata_kvm_4.19.x]: I explicitly chose these, and explicitly disabled the =n ' s

**Comment by grahamwhaley** [comment]
I'll contemplate adding a fragment for this to the pending fragment PR, at the risk of stalling its merge further...

**Pull request merged**